Gobabygo/Gobabygo.ino

```diff
@@ -56,15 +56,13 @@ void loop() {
   previousTime = currentTime;
   
   float rawJoyXValue = analogRead(joyStickXPin);
-  joyXVal = analogRead(joyStickXPin);
-  joyXVal = 2.0/1023.0 * joyXVal - 1;
+  joyXVal = 2.0/1023.0 * analogRead(joyStickXPin) - 1;
```

Comment:
This was almost what I was suggesting. Here is what I meant to decrease the number of calls to analogRead:

```joyXVal = 2.0/1023.0 * rawJoyXValue - 1;```

Same for the ```joyYVal``` case.